tests/dbshell/test_postgresql.py

```diff
@@ -14,15 +14,19 @@ def _run_it(self, dbinfo):
         That function invokes the runshell command, while mocking
         subprocess.run(). It returns a 2-tuple with:
         - The command line list
-        - The the value of the PGPASSWORD environment variable, or None.
+        - The list of values of the PGP% environment variables, or {}.
         """
-        def _mock_subprocess_run(*args, env=os.environ, **kwargs):
+        def _mock_subprocess_run(*args, env=os.environ, **_kwargs):
             self.subprocess_args = list(*args)
-            self.pgpassword = env.get('PGPASSWORD')
+            # Get all postgres related env
+            # Full list can be found at https://www.postgresql.org/docs/current/libpq-envars.html
```

Comment:
I don't see much value in this comment, please remove it.

Reply:
Ok! Done.